### PR TITLE
change header color, primary buttons, and top border to green for ETC

### PR DESCRIPTION
**Pull request by willpiers**
These changes were made in response to an [ETC Bounty](https://dexaran.github.io/EthereumCommonwealth/Bounties/)

This PR updates header, top border, and all primary buttons to reflect that the user is interacting with the ETC node.

screen shot 2017-07-09 at 7 16 27 pm

### CODE REVIEW

**Comment by willpiers** [comment]
The only files I changed were `header.tpl`, `index.html`, `viewCtrl.js`, and `etherwallet-custom.less`

Not sure if I was supposed to commit the rest. Let me know and I will make another PR to add clarifying info to the README

File: app/includes/header.tpl [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 <body>
 
-<header class="{{curNode.name}} {{curNode.service}} {{curNode.service}} nav-index-{{gService.currentTab}}" aria-label="header" ng-controller='tabsCtrl' >
+<header class="{{curNode.name}} {{curNode.service}} nav-index-{{gService.currentTab}}" aria-label="header" ng-controller='tabsCtrl' >
```
Comment (willpiers) [inline app/includes/header.tpl:34]: This just looked unnecessary to me, so I removed it.

File: app/styles/etherwallet-custom.less [hunk 2]
```diff
@@ -574,5 +577,9 @@ header.Custom  {
   }
 }
 
-
-
+.container.ETC {
+  .btn.btn-primary {
+    border: none;
+    background-color: #3DB753;
```
Comment (willpiers) [inline app/styles/etherwallet-custom.less]: This is the same as `@brand-etc` - I'll push a change

File: app/includes/header.tpl (hunk 1 above)
Comment (tayvano) [reply app/includes/header.tpl:34]: Yes - is unnecessary.

**Comment by tayvano** [comment]
FYI - Cody's got this in v3.10 which I aim to test tonight and deploy tomorrow.

**Comment by willpiers** [comment]
Awesome, looking forward to it. Make sure Cody claims [the bounty](https://dexaran.github.io/EthereumCommonwealth/Bounties/#marker-bounty-1)!

**Comment by Dexaran** [comment]
No one has claimed the bounty yet. You've did exactly what I was talking about. I'm ready to pay the bounty reward as soon as changes would appear at the web version of MEW.

**Comment by Dexaran** [comment]
Spotted a mistake. Button color doesn't change when you switch the node.

**Comment by alexbarnsley** [comment]
@Dexaran want me to take a look at your last comment as it's been over a month?

**Comment by kvhnuke** [comment]
I believe this is outdated, please open a new PR

**Pull request closed without merging**